packages/wpcom-checkout/src/checkout-labels.ts

```diff
@@ -93,5 +93,9 @@ export function getLabel( product: ResponseCartProduct ): string {
 	if ( product.meta && ( isDomainProduct( product ) || isDomainTransfer( product ) ) ) {
 		return product.meta;
 	}
+
+	if ( hasCheckoutVersion( '2' ) && isPlan( product ) ) {
+		return product.product_name.replace( 'WordPress.com ', '' ).concat( ' plan' );
```

Comment:
I understand why you did this, but it's probably not a good idea to directly manipulate translated strings like this. While `wordpress.com` itself is probably always in english, the space that comes after it might not be, and `' plan'` is not being translated at all.

I think we should just remove this hack for now and show the full title of each product. Those come from the server and if we want to change them we should change them there.